src/components/sales-report/index.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react'
-import SalesTable from './SalesTable'
-import { connect } from 'react-redux'
-import selectors from '../../redux/selectors'
+import SalesTable from './salesTable'
+import * as ReactRedux from 'react-redux'
+import * as Redux from 'redux'
+import * as ReactReduxFirebase from 'react-redux-firebase'
```

Comment:
Let's first import 3rd party packages and do that in a separate block. Also sort every block alphabetically. Like so:
```js
import React from 'react'
import * as ReactRedux from 'react-redux'
import * as ReactReduxFirebase from 'react-redux-firebase'
import * as Redux from 'redux'

import SalesTable from './salesTable'
```